operator/v1/types_network.go

```diff
@@ -65,6 +65,13 @@ type NetworkSpec struct {
 	// +optional
 	DeployKubeProxy *bool `json:"deployKubeProxy,omitempty"`
 
+	// enableNetworkDiagnostics specifies whether or not periodic
+	// network connectivity checks should be enabled or not. If unset, this
+	// property defaults to 'true' and network diagnostics is enabled.
+	// +optional
+	// +kubebuilder:default:=true
+	EnableNetworkDiagnostics *bool `json:"enableNetworkDiagnostics,omitempty"`
```

Comment:
If you want to enable that by default, and you rarely will turn this off, why not going w/o pointer and call it `DisableNetworkDiagnostics` with default being true. That seems the most reasonable approach.

Reply:
I guess you meant to default being false, we want it to have it enabled by default

Reply:
Yes of course :sweat_smile:

Reply:
done